core/src/main/java/com/netease/arctic/trace/AmsTableTracer.java

```diff
@@ -171,7 +171,7 @@ public void commit() {
       commitMeta.setSchemaUpdateMeta(ddlCommitMeta);
       update = true;
     }
-    if (this.properties != null) {
+    if (this.properties != null && Constants.INNER_TABLE_BASE.equals(innerTable)) {
```

Comment:
Could you please combine the `Constants.INNER_TABLE_BASE.equals(innerTable)` into one condition with 163 line?
```
if (Constants.INNER_TABLE_BASE.equals(innerTable)) {
  update = true;
}
```

Reply:
When there is datafile commited both baseTable and changeTable should commit tableChange, so the we can't set `update = true`  when Constants.INNER_TABLE_BASE.equals(innerTable)

Reply:
What is the condition that there is datafile commited both baseTable and changeTable should commit tableChange? IMO, whether the properties are null couldn't prove that there is datafile commited both baseTable and changeTable.

Reply:
yes you are right, the condition of tableChange is  datafile commited both baseTable and changeTable should commit tableChange.  the condition  properties are null is just for table just alter properties but not commit datafile. AmsTableTracer can commit many kinds of tracer data like alter properties, alter schema, commit data.